src/Uno.Extensions.Templates/content/unoapp-extensions/azure/steps/android.yml

```diff
@@ -0,0 +1,116 @@
+parameters:
+- name: major
+  type: string
+  default: '0'  
```

Comment:
Can we have an approach to versioning so that all platforms end up with same version?